src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data.py

```diff
@@ -411,28 +411,35 @@ def determine_needed_experience_configs(bind):
     """
 
     notice_query = """select id, name, regions, displayed_in_overlay, displayed_in_privacy_center, notice_key from privacynotice;"""
-    experience_config_query = """SELECT id, component, disabled, accept_button_label, banner_enabled, description, privacy_preferences_link_label, privacy_policy_link_label, privacy_policy_url, save_button_label, title, banner_description, banner_title, reject_button_label, acknowledge_button_label from privacyexperienceconfig;"""
+    experience_config_query = """SELECT id, component, disabled, accept_button_label, description, privacy_preferences_link_label, privacy_policy_link_label, privacy_policy_url, save_button_label, title, banner_description, banner_title, reject_button_label, acknowledge_button_label from privacyexperienceconfig;"""
 
     existing_notices: ResultProxy = bind.execute(text(notice_query))
     for existing_notice in existing_notices:
         notice_key = existing_notice["notice_key"]
         component_mappings = NOTICES_TO_EXPERIENCE_CONFIG.get(notice_key, [])
         if component_mappings:
             for component_mapping in component_mappings:
-                regions_to_add = set(existing_notice["regions"]).copy()
-
-                # remove GB subregions and replace with top-level `gb` region!
-                if not GB_REGIONS_TO_REMOVE.isdisjoint(
-                    regions_to_add
-                ):  # if we have GB regions to remove
-                    regions_to_add.difference_update(
-                        GB_REGIONS_TO_REMOVE
-                    )  # remove the GB regions
-                    regions_to_add.update(GB_REGIONS_TO_ADD)  # add the GB region
-
-                # ignore any canada regions in migration - they are not migrated,
-                # but instead handled by a net-new OOB experience
-                regions_to_add.difference_update(CA_REGIONS_TO_IGNORE)
+                if (
+                    component_mapping.get(ComponentType.Overlay)
+                    == DefaultExperienceConfigTypes.CANADA_BANNER_MODAL
+                ):
+                    # If this is a Canadian Banner and Modal, skip updating regions, and just use OOB
+                    regions_to_add = set()
```

Comment:
This is not very pretty here - special casing so the canadian banner and modal doesn't have notice regions on it, but using OOB regions instead.